src/RockblockCommand.hpp

```diff
@@ -108,6 +110,8 @@ class FaultOverrideCommand : public RockblockCommand
     {
         if (FaultInterface::opcode_lookup.find(f_opcode) != FaultInterface::opcode_lookup.end()) {
             fault = FaultInterface::opcode_lookup[f_opcode];
+        } else {
+            fault = nullptr;
         }
     };
 
```

Comment:
If assigning the fault to nullptr, you would need to add an exit condition (or a check) to execute for if its not valid (AKA if its nullptr) otherwise could get nullptr exception since f_arg_1 and f_arg2 could still be set.

(i know technically its execute function won't be called if it doesn't pass is_valid because it won't be added to processed_commands, but I think it is safer to add a gate in execute anyways - similar to how its done in the SFROverrideCommand::execute function or by adding an if its nullptr{return} clause first)

Reply:
Oh lol Eric beat me to this haha

Reply:
Because of the potential null pointer exception I agree we should add it

Reply:
Plus we use isValid in SFROverride’s execute